projects/storefrontlib/src/cms-components/misc/site-context-selector/language-currency.component.ts

```diff
@@ -0,0 +1,11 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'cx-language-currency-selector',
+  template: `
+    <cx-site-context-selector context="LANGUAGE"></cx-site-context-selector>
+    <cx-site-context-selector context="CURRENCY"></cx-site-context-selector>
+  `,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class LanguageCurrencyComponent {}
```

Comment:
I would consider some simple unit test for this component, at least for sake of consistency and code coverage.